.ci/Jenkinsfile

```diff
@@ -71,6 +71,12 @@ pipeline {
           expression { return params.FORCE_SEND_PR }
         }
       }
+      environment {
+        // GIT_PREVIOUS_SUCCESSFUL_COMMIT might point to a local merge commit instead a commit in the
+        // origin, then let's use the target branch for PRs and the GIT_PREVIOUS_SUCCESSFUL_COMMIT for
+        // branches.
+        COMMIT_FROM = """${isPR() ? "origin/${env.CHANGE_TARGET}" : "${env.GIT_PREVIOUS_SUCCESSFUL_COMMIT}" """
```

Comment:
Is this behaviour specific to this repo? I wonder if it could be applied everywhere directly in the library 🤔

Reply:
I'm skeptical to change the library though. The step uses git instead of using the git previous success variable that's populated from Jenkins

Reply:
Ok, that sounds reasonable